src/components/CustomInput.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, TextInput, View } from 'react-native';
+
+interface CustomInputProps {
+  label: string;
+  secureTextEntry?: boolean;
+  onSubmit: (text: string) => void;
+}
+
+const CustomInput: React.FC<CustomInputProps> = ({
+  label,
+  secureTextEntry,
+  onSubmit,
+}) => {
+  const [text, setText] = useState('');
```

Comment:
jeśli już byś chciał użyć `useState`, co w tym przypadku jest zbędne, to na przyszłość zauważ, że w takim przypadku jak tutaj mamy czasami do czynienia z inputami, które mają domyślnie wpisaną jakąś wartość. Wówczas `useState` powinien przyjąć tą domyślną wartość jako wartość początkową, a nie pustego stringa